curvegrid/multibaas-for-google-sheets
# [#19] Refactor menus

Part of #19 

- Add menus to set deployment ID and API key.
- Refactor according to menu changes
- Update tests

**Manual Tests**

- Create a new spreadsheet
- Go to `Add-ons > Get add-ons`
- Type `multibaas` in search box
- Install (wait for installing)
- Go to `Add-ons > MultiBaas for Google Sheets > Set Deployment ID` and set it
- Go to `Add-ons > MultiBaas for Google Sheets > Set API Key` and set it
- Call functions and check all functions work well
